fs/zbd_zenfs.cc

```diff
@@ -428,12 +428,13 @@ void ZonedBlockDevice::LogGarbageInfo() {
   // the result to be precise.
   int zone_gc_stat[12] = {0};
   for (auto z : io_zones) {
-    if (z->IsEmpty()) {
-      zone_gc_stat[0]++;
+    if (z->IsBusy() || !z->Acquire()) {
```

Comment:
Please drop the IsBusy call, it is checked by Acquire() anyway.

Reply:
Also the code will not work as intended, because `IsBusy()` uses relaxed memory ordering. You have to obtain the lock with `Acquire()` before using `IsBusy` if you want proper memory ordering, otherwise there are no guarantees about ordering of the writes and reads of the busy flag.